framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -500,6 +500,31 @@ private DefaultSet defaultsAt(final Element elt) {
         }
     }
 
+    public boolean applyUnannotatedDefaults(final Element annotationScope) {
+        boolean annotatedForThisChecker = isElementAnnotatedForThisChecker(annotationScope);
+
+        if (unannotatedDefaults.size() > 0) {
+            // TODO: I would expect this:
+            //   atypeFactory.isFromByteCode(annotationScope)) {
```

Comment:
Are you saying that the condition in the below if statement should be replaced by  atypeFactory.isFromByteCode(annotationScope))?